nevergrad/functions/pcse/pcse.py

```diff
@@ -0,0 +1,143 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+"""
+Approximate crop Simulation
+Based on
+https://raw.githubusercontent.com/purdue-orbital/pcse-simulation/master/Simulation2.py
+"""
+
+
+import numpy as np
+from ..base import ArrayExperimentFunction
+
+# pylint: disable=too-many-locals,too-many-statements
+
+
+class Pcse(ArrayExperimentFunction):
+    def __init__(self) -> None:
+        import yaml
+        import pandas as pd
```

Comment:
standard imports should be on top, pcse can be kept inline to avoid issues

Reply:
Done.

Reply:
> standard imports should be on top, pcse can be kept inline to avoid issues

Done.